### PR TITLE
Added method to set label on the "Use" button on Still shot Preview

### PR DESCRIPTION
**Pull request by wscheffer**

### CODE REVIEW

**Comment by mowcixo** [comment]
I think it'd be better to replace the name of current ```labelUseVideo``` method, instead of creating a new method and property... something like ```labelConfirm```, let's see what people think...

**Comment by wscheffer** [comment]
Would also prefer if it only had one method, would suggest adding labelConfirm and mark labelUseVideo as @Deprecated

File: library/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -4,6 +4,7 @@
     <string name="mcam_permissions_needed">Permissions Needed</string>
     <string name="mcam_video_perm_warning">Permission to access your microphone and camera is needed, the camera will now close.</string>
     <string name="mcam_retry">RETRY</string>
+    <string name="mcam_confirm">CONFIRM</string>
```
Comment (afollestad) [inline library/src/main/res/values/strings.xml]: I don't think adding a third string is a good idea. By default, it should use `Use Video` or `Use Picture` based on whether stillshot is enabled. @wscheffer @mowcixo

Comment (wscheffer) [inline library/src/main/res/values/strings.xml]: Ok will do

Comment (wscheffer) [inline library/src/main/res/values/strings.xml]: Issue is that BaseCaptureActivity needs a default value for getIntent().getIntExtra(CameraIntentKey.LABEL_CONFIRM, R.string.mcam_confirm);
so we need something more generic that will work for Video and Picture

Comment (afollestad) [inline library/src/main/res/values/strings.xml]: You could add an integer parameter to `labelUseVideo(int)` which would indicate a default value. The video or camera fragments could provide a default through that parameter.

Comment (afollestad) [inline library/src/main/res/values/strings.xml]: Sorry, I meant `labelConfirm`

**Comment by wscheffer** [comment]
Made some changes if you can check if it would be sufficient @afollestad

**Comment by mowcixo** [comment]
I find it perfect now... just for the changes in ```MainActivity.java```, I'd left it with defaults.

**Comment by wscheffer** [comment]
I guess you can but BaseCaptureActivity:522 shouldn't compensate for this you should not need to set the default text

```java
  public int labelConfirm() {
        return getIntent().getIntExtra(CameraIntentKey.LABEL_CONFIRM, useStillshot() ? R.string.mcam_use_stillshot : R.string.mcam_use_video);
    }
```

**Comment by afollestad** [comment]
Thanks guys!

**Pull request merged**